high_card.rb

```diff
@@ -1,5 +1,53 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
+# Make a deck
+def deck_of_cards
+    ranks = %w(2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K', 'A')
+    suits = %w(hearts, diamonds, clubs, spades)
+    deck = []
+
+    ranks.each_with_index do |r, i|
+        suits.each do |s|
+            deck.push(ranks: r, suits: s, score: i)
+        end
+    end
+#shuffle deck
+    deck.shuffle
+end
+
+# players:
+deck = deck_of_cards
 players = []
+
+# get names:
+loop do
```

Comment:
For some reason, this just loops like 26 times for me until asking for input...did it work fine for you, looping once, asking for input, then breaking if you typed 'ok'?